index.php

```diff
@@ -47,5 +47,6 @@
       </div>
 <?php } 
 
-Template::render_footer();
-}
\ No newline at end of file
+Template::render_footer(false,COPYRIGHT_TEXT);

```

Comment:
Passing a named constant seems wrong to me... Why not simply use it in the function directly?

Reply:
Will fix

Reply:
Fixed